run_all_mac.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+export hse=$(pwd)
+#newtab='tell application "System Events" to keystroke "t" using command down'
+echo $hse
+
+export NOTIFY_API_KEY="${NOTIFY_API_KEY:-test-key}"
+export GOOGLE_ANALYTICS_TRACKING_ID="test-key"
+echo 'Notify key being used is: ' $NOTIFY_API_KEY
+
+#osascript -e 'tell application "Terminal"' -e 'activate' -e 'to do script cd $hse/htbhf-applicant-web-ui; git status; npm start; pause.sh' -e 'end tell'
+osascript -e "tell application \"Terminal\" to do script \"cd $hse/htbhf-applicant-web-ui; git status; npm start; pause.sh\""
```

Comment:
Is there any way this can be improved to run in a tabbed terminal window? This opens a new terminal window for each process which is pretty intense...

Reply:
This is work in progress, unfortunately there isn't an easy way to run the command in a new tab